.github/workflows/create-prs-to-update-vcs-repositories.yaml

```
@@ -0,0 +1,31 @@
+name: create-prs-to-update-vcs-repositories
+
+on:
+  schedule:
+    - cron: 0 0,6,12,18 * * *
+  workflow_dispatch:
+
+jobs:
+  create-version-update-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Generate GitHub App token
+        id: generate-token
+        uses: tibdex/github-app-token@v2
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+
+      - name: Create PRs to update VCS repositories
+        uses: autowarefoundation/autoware-github-actions/create-prs-to-update-vcs-repositories@v1
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          repo_name: autowarefoundation/autoware
+          parent_dir: .
+          targets: major minor
```

We would like to also update the patch version. What do you think?

```suggestion
          targets: any
```

---

@youtalk Thank you for your review!
@mitsudome-r @xmfcx I agree with Yutaka-san's proposal. Shall we go ahead with three version update supports if it is okay?

---

@mitsudome-r @xmfcx The reason why `major` and `minor` are specified is [the conclusion](https://github.com/autowarefoundation/autoware-github-actions/pull/320#issuecomment-2357480736) of the software working group two weeks ago :+1: . But having the patch update too looks useful because the pace of update is vary in each repository.

Hopefully this is useful to clarify the background.

---

Yes, I'm okay with the proposal :+1:

---

@mitsudome-r Thank you! Now I applied the change.

@youtalk As the `any` option creates only one PR for any newer version, I applied `major minor patch` :+1: